modules/tinymce/src/core/main/ts/Rtc.ts

```diff
@@ -272,6 +273,45 @@ const makeRtcAdaptor = (tinymceEditor: Editor, rtcEditor: RtcRuntimeApi): RtcAda
   };
 };
 
+const makeNoopAdaptor = (): RtcAdaptor => ({
+  undoManager: {
+    beforeChange: () => {},
+    addUndoLevel: () => null,
+    undo: () => null,
+    redo: () => null,
+    clear: () => {},
+    reset: () => {},
+    hasUndo: () => false,
```

Comment:
Likewise `Fun.never` here instead please.

- [ ] `Fun.never` instead of `() => false`